__tests__/app.test.js

```diff
@@ -0,0 +1,29 @@
+const app = require('../db/app')
+const request = require("supertest");
+const db = require("../db/connection");
+const categoryData = require("../db/data/test-data/categories");
+const commentData = require('../db/data/test-data/comments');
+const reviewData = require('../db/data/test-data/reviews');
+const userData = require('../db/data/test-data/users');
+const seed = require("../db/seeds/seed");
+
+beforeEach(() => {
+    return seed({ categoryData, commentData, reviewData, userData });
+  });
+  
+  afterAll(() => {
+    return db.end();
+  });
+
+describe("app tests", () => {
+    test("status 200", () => {
+        return request(app).get("/api/").expect(200);
+    })
+    test("returns categories, containing properties of slug and description", () => {
+        return request(app).get("/api/categories/").expect(200).then((response) => {
+            //console.log(response.body)
+           expect(response.body.categories[0]).toHaveProperty("slug");
+           expect(response.body.categories[0]).toHaveProperty("description");
+        })
+    })
+})
```

Comment:
You can also use a forEach for this to make sure that every object in the returned objects contains a slug and a description using 'objectContaining' and checking the length of the returned array. This will come in very handy as you go along.